coins.json

```diff
@@ -817,5 +817,25 @@
         "explorer": "https://terra.stake.id/?#/tx/",
         "url": "https://terra.money",
         "rpcNodeInfo": "https://github.com/terra-project/core"
+    },
+    {
+        "id": "monacoin",
+        "name": "Monacoin",
+        "symbol": "MONA",
+        "decimals": 8,
+        "blockchain": "Bitcoin",
+        "derivationPath": "m/44'/22'/0'/0/0",
+        "curve": "secp256k1",
+        "publicKeyType": "secp256k1",
+        "p2pkhPrefix": 50,
+        "p2shPrefix": 55,
+        "hrp": "mona",
+        "publicKeyHasher": "sha256ripemd",
+        "base58Hasher": "sha256d",
+        "xpub": "xpub",
+        "xprv": "xprv",
+        "explorer": "https://blockbook.electrum-mona.org/tx/",
```

Comment:
This is the blockbook node, what block explorer does normal user use? https://bchain.info/MONA/ ?

Reply:
Yeah, that's probably the main explorer that people use. (There isn't an officially sanctioned one as far as I know, and the Insight-based one appears to be down...)

Reply:
can you provide blockbook node for us to use? and make sure you added the implementation on blockbook repo.

Reply:
@lae can you message me on telegram: @vikmeup? wanted to discuss final integration

Reply:
@vikmeup I sent you a message - it would be preferable if I could get you into contact with WakiyamaP instead of directly with me, but I can act as mediator. (my telegram seems to have been deleted after neglecting it for so long - so it's a new account.)